nixos/modules/services/web-apps/dokuwiki.nix

```diff
@@ -0,0 +1,281 @@
+{ config, lib, pkgs, ... }:
+
+let
+
+  inherit (lib) mkEnableOption mkForce mkIf mkMerge mkOption recursiveUpdate types;
+
+  cfg = config.services.dokuwiki;
+  fpm = config.services.phpfpm.pools.dokuwiki;
+
+  user = config.services.nginx.user;
+  group = config.services.nginx.group;
+
+  pkg = pkgs.dokuwiki;
+
+  dokuwikiAclAuthConfig = pkgs.writeText "acl.auth.php" ''
+    # acl.auth.php
+    # <?php exit()?>
+    #
+    # Access Control Lists
+    #
+    ${toString cfg.acl}
+  '';
+
+  dokuwikiLocalConfig = pkgs.writeText "local.php" ''
+    <?php
+    $conf['savedir'] = '${cfg.stateDir}';
+    ${toString cfg.extraConfig}
+  '';
```

Comment:
Missing closing tag?

Reply:
That's exactly how dokuwiki generates these files: https://github.com/splitbrain/dokuwiki/blob/master/conf/local.php.dist